### PR TITLE
Rake commands to ping DynamoDB and to create tables from the models

### PR DESCRIPTION
**Pull request by NrBT**
I've created a couple of simple rake commands for Rails to: 
1. test the connection to the DynamoDB (local or remote) using the current Dynamoid.configuration settings
2. Create the tables in advance of using them which kinda works around the lag Dynamo has when creating tables

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -4,8 +4,6 @@
 $LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '..', 'lib'))
 $LOAD_PATH.unshift(File.dirname(__FILE__))
 
-MODELS = File.join(File.dirname(__FILE__), "app/models")
```
Comment (NrBT) [inline spec/spec_helper.rb]: This line is duplicated a little later on

File: lib/dynamoid/document.rb [hunk 2]
```diff
@@ -13,7 +13,7 @@ module Document
       self.read_only_attributes = []
       self.base_class = self
 
-      Dynamoid.included_models << self
+      Dynamoid.included_models << self unless Dynamoid.included_models.include? self
```
Comment (NrBT) [inline lib/dynamoid/document.rb]: I saw a case where the same class had been added to the Dynamoid.included_models more than once. I couldn't find a way to reproduce it so haven't written a test proof.

**Comment by pboling** [comment]
Please rebase on latest master - Spec suite is fixed!!!

**Comment by pboling** [comment]
@NrBT still interested in this?

**Pull request merged**